SiddhiBenchmark/FilterBenchmark/src/main/java/Main.java

```diff
@@ -1,15 +1,10 @@
-import java.io.BufferedReader;
```

Comment:
Lets have a meaningful name than Main.java

Reply:
Changed the class name to Benchmark